app/pages/preview/createPreviewPageContext.test.js

```diff
@@ -23,4 +23,32 @@ describe('createPreviewPageContext', () => {
 
     expect(context).toEqual(expectedContext);
   });
+
```

Comment:
Another test for when the document name doesn't exist and then perhaps another if there is more than just one?